src/tempo_cluster.py

```diff
@@ -180,18 +174,12 @@ class DatabagModel(BaseModel):
             extra="ignore",
             # Allow instantiating this class by field name (instead of forcing alias).
             populate_by_name=True,
-            # Custom config key: whether to nest the whole datastructure (as json)
-            # under a field or spread it out at the toplevel.
-            _NEST_UNDER=None,
         )  # type: ignore
         """Pydantic config."""
 
         @classmethod
         def load(cls, databag: MutableMapping):
             """Load this model from a Juju databag."""
-            nest_under = cls.model_config.get("_NEST_UNDER")
```

Comment:
uhm, why did you remove the nest_under logic from the whole object?

Reply:
@sed-i raised a comment if we still need that since it might have been introduced due to old pydantic limitation.
Seems that with the pydantic version that gets used in this charm, things work normally without this object and its logic.